src/Oscoin/Node/Options.hs

```diff
@@ -58,6 +60,14 @@ nodeOptionsParser cps = Options
        <> help "Do not generate empty blocks"
        <> showDefault
         )
+    <*> option (map (* 1000000000) auto)
+        ( long "empty-block-time"
+       <> help "Wait approx. this many seconds between empty blocks. Has no \
+               \effect when running with production settings."
+       <> metavar "SECONDS"
+       <> value 1
+       <> showDefault
+        )
```

Comment:
Should we prefix this option to communicate that it affects the lenient consensus? Eg. `lenient.empty-block-time` or something?

Reply:
We can also punt on this and revisit our whole cli options scheme at some point in the future :man_shrugging: